### DISCUSSION TITLE
Не пропускает API ключ

### DISCUSSION THREAD

**Discussion by offon** [IATkachenko/HA-YandexWeather#52] [Q&A] [answered]
День добрый, подскажите пожалуйста интеграция у все нормально работает? При подключении вставляю API ключ как указано в инструкции и получаю соответствующую ошибку "Не могу получить данные. Проверьте корректность API-ключа в кабинете разработчика и убедитесь что выбран тариф 'Погода для сайта'. Может потребоваться до 30 минут, чтобы изменения настроек ключа вступили в силу на стороне Яндекса." Ключ валидный 100% и времени уже 2 часа прошло. Подскажите куда копать

**Comment by offon** [comment, marked as answer]
Отбой, ключ прошел спустя 3 часа....

**Reply by IATkachenko** [reply]
3 часа, новый рекорд!

**Comment by FLieR-13** [comment]
Привет, тоже API не пропускает.. ошибка (Не могу получить данные. Проверьте корректность API-ключа в кабинете разработчика и убедитесь что выбран тариф 'Погода для сайта'. Может потребоваться до 30 минут, чтобы изменения настроек ключа вступили в силу на стороне Яндекса.) Уже прошло больше 3х часов =)) Яндекс.Погода вообще работает?

**Comment by FLieR-13** [comment]
Вообщем не работает ваше приложение, API не пропускает, прошло уже больше 3х дней..

**Comment by IATkachenko** [comment]
@FLieR-13, откройте, пожалуйста, новое issue -- будем разбираться что происходит...
Вангую что при создании ключа сразу выбрали "погода для сайта", а не тестовый с переключением, как написано в документации...

**Reply by FLieR-13** [reply]
Делал тестовый, в итоге попробовал 2й апи сделать, в итоге яндекс заблокировал - API Яндекс.Погоды Заблокировано - на вопросы не отвечают..

**Comment by Vooovan** [comment]
не работает api ключ. тоже самое. 

Не могу получить данные. Проверьте корректность API-ключа в кабинете разработчика и убедитесь что выбран тариф 'Погода для сайта'. Может потребоваться до 30 минут, чтобы изменения настроек ключа вступили в силу на стороне Яндекса.

прошли сутки с получения

**Reply by IATkachenko** [reply]
завести #95 -- отличное решение, разбираемся там.

**Comment by Andrey-simf** [comment]
Добрый вечер та же ошибка. 
"Не могу получить данные. Проверьте корректность API-ключа в кабинете разработчика и убедитесь что выбран тариф 'Погода для сайта'. Может потребоваться до 30 минут, чтобы изменения настроек ключа вступили в силу на стороне Яндекса."
Сейчас в кабинете разработчика нет API Яндекс погоды, у них отдельный раздел. в начале нечаянно создал для бизнеса тестовый, после создал новый аккаунт в Яндексе и создал погоду для умного дома, для сайта негде не могу найти. Прошло 2 дня API не тестовый бизнес, не для умного дома не подходят. Куда копать?

**Reply by IATkachenko** [reply]
> Добрый вечер та же ошибка. "Не могу получить данные. Проверьте корректность API-ключа в кабинете разработчика и убедитесь что выбран тариф 'Погода для сайта'. Может потребоваться до 30 минут, чтобы изменения настроек ключа вступили в силу на стороне Яндекса." Сейчас в кабинете разработчика нет API Яндекс погоды, у них отдельный раздел. в начале нечаянно создал для бизнеса тестовый, после создал новый аккаунт в Яндексе и создал погоду для умного дома, для сайта негде не могу найти. Прошло 2 дня API не тестовый бизнес, не для умного дома не подходят. Куда копать?

Интеграция пока не поддерживает V3 API, а все новые ключи для этой версии API.
Stay tuned, https://github.com/IATkachenko/HA-YandexWeather/issues/127 обязательно сойдется

**Comment by 0dminka** [comment]
https://github.com/yandex/pogoda-home-assistant
вот от яндекса офф интеграция рабочая
